pkg/reconciler/shared/tektonconfig/tektonconfig.go

```diff
@@ -138,3 +152,64 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tc *v1alpha1.TektonConfi
 
 	return nil
 }
+
+func (r *Reconciler) createOperatorVersionConfigMap(tc *v1alpha1.TektonConfig) error {
+	koDataDir := os.Getenv(common.KoEnvKey)
+	operatorDir := filepath.Join(koDataDir, "info")
+
+	if err := common.AppendManifest(&r.manifest, operatorDir); err != nil {
+		return err
+	}
+
+	manifest, err := r.manifest.Transform(
+		mf.InjectNamespace(tc.GetSpec().GetTargetNamespace()),
```

Comment:
could you make sure that he ownerReference is also set to TektonConfig instance. So that we can make sure this config map is cascade-deleted when we delete the instance (name:config) of TektonConfig